scalafix-tests/unit/src/test/scala/scalafix/tests/cli/CliGitDiffTests.scala

```diff
@@ -0,0 +1,257 @@
+package scalafix.tests.cli
+
+import java.lang.ProcessBuilder
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.PrintStream
+import java.nio.charset.StandardCharsets
+import java.nio.charset.StandardCharsets
+import java.nio.file.{Files, Paths, Path, StandardOpenOption}
+
+import org.scalactic.source.Position
+import org.scalatest._
+import org.scalatest.FunSuite
+
+import scala.collection.immutable.Seq
+import scala.util._
+import scalafix.cli
+import scalafix.internal.cli.CommonOptions
+import scalafix.testkit.DiffAssertions
+
+class CliGitDiffTests() extends FunSuite with DiffAssertions {
+  gitTest("it should handle addition") { (fs, git, cli) =>
```

Comment:
Name this test "addition"? The convention in scalafix tests is to try keep test names as concise as possible

Reply:
with utest it's Possible to run a single test by name, which makes sentences annoying to work with